src/main/scala/sample/reactivekafka/KafkaReaderCoordinator.scala

```diff
@@ -41,19 +41,46 @@ class KafkaReaderCoordinator(mat: Materializer, topicName: String) extends Actor
       .kafkaOffsetsStorage()
       .commitInterval(1200 milliseconds)
     )
+
     log.debug("Starting the reader")
-    Source(consumerWithOffsetSink.publisher)
-      .map(processMessage)
-      .withAttributes(ActorAttributes.supervisionStrategy(processingDecider))
-      .to(consumerWithOffsetSink.offsetCommitSink).run()
```

Comment:
I've been thinking about the manual committing issue.
I don't think the manual commit would be useful in this case because it's emitting to multiple sinks.
The previous example was manually committing every message so I don't think it's doing any worse than it was previously doing.